rlclientlib/logger/async_batcher.h

```diff
@@ -136,10 +145,12 @@ namespace reinforcement_learning { namespace logger {
   int async_batcher<TEvent, TSerializer>::fill_buffer(
                                                       std::shared_ptr<utility::data_buffer>& buffer,
                                                       size_t& remaining,
-                                                      api_status* status)
+                                                      api_status* status,
+                                                      unsigned int* original_count)
   {
     TEvent evt;
     TSerializer<TEvent> collection_serializer(*buffer.get(), _batch_content_encoding, _shared_state);
+    if (_events_counter_status == events_counter_status::ENABLE) { _buffer_start_event_number = _buffer_end_event_number; }
```

Comment:
Can you move 
_buffer_start_event_number = _buffer_end_event_number;
in line 168 below?

Reply:
updated